### PR TITLE
Use FSL CSS style sheet

### PR DESCRIPTION
**Pull request by TomMaullin**
This branch mimics the design of the actual FSL results viewer by using the stylesheet found within FSL in order to address issue #6 . It also includes code allowing the embedding of images in order to embed the FSL background and logo images and therefore serves as a start towards resolving issue #8 (I will fully address this issue once PR #9 is merged!). @cmaumet Please let me know what you think of this!

Note: This branch assumes FSL is installed in order to locate the FSL stylesheet. I am not sure if Travis testing currently allows for this.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from setuptools import setup, find_packages
+import pip
```
Comment (cmaumet) [inline setup.py]: Can we remove this `pip` import?

Comment (cmaumet) [inline setup.py]: Can we remove this `pip` import?

File: nidmviewerfsl/pageStyling.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+#======================================================================
+# This file contains a collection of small functions for reading in and
+# encoding images as well as the CSS styling for FSL output pages. It
+# assumes FSL is installed.
```
Comment (cmaumet) [inline nidmviewerfsl/pageStyling.py]: @TomMaullin: now that we actually depend on FSL, could you update the README to state this explicitly?

File: nidmviewerfsl/pageStyling.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+#
+# Author: Tom Maullin (04/12/2017)
+#======================================================================
+import subprocess
```
Comment (cmaumet) [inline nidmviewerfsl/pageStyling.py]: Is the `subprocess` import needed?

**Review by cmaumet — APPROVED** [review]
Thank you @TomMaullin. The new output is very nice!  

This update looks good to me. Once the small comments are addressed, I am happy for you to merge.

**Comment by cmaumet** [comment]
Hi @TomMaullin, now that you have merged #9, you can see that this PR can no longer be automatically merged (cf. message below "This branch has conflicts that must be resolved"). 

This happens when two updates (here #9 and the current PR) are proposed on the same portion of code and git needs you to decide how the two set of changes should be merged. To do so (i.e. 'resolve the conflict'), you can follow GitHub instructions above either using the web editor or using command line on your machine. 

Can you try and solve the conflict to re-enable the possibility to merge this PR? Let me know if you need more info!

**Comment by TomMaullin** [comment]
All feedback should now be resolved.

**Pull request merged**